sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusClientBuilder.java

```diff
@@ -56,6 +56,24 @@
 /**
  * The builder to create Service Bus clients:
  *
+ * <p><strong>Instantiate sync receiver</strong></p>
+ * {@codesnippet com.azure.messaging.servicebus.receiver.sync.client.instantiation}
+ *
+ * <p><strong>Instantiate sync sender</strong></p>
```

Comment:
I think showing how to instantiate one of each type of client is enough.
* Receiver
* Sender
* SessionReceiver
* Processor

Having an exhaustive matrix adds more cognitive load.